### PR TITLE
Replaced MIT license header with Xcode header #19

### PR DESCRIPTION
**Pull request by vknabel**
This PR replaces the MIT headers in `{PROJECT}.swift` and `{PROJECT}Tests.swift` with headers Xcode would generate.

But I am not sure how to handle different licenses.

### CODE REVIEW

File: main.swift [hunk 1]
```diff
@@ -65,6 +65,7 @@ class StringReplacer {
     private let authorEmail: String
     private let gitHubURL: String
     private let year: String
+    private let today: String
```
Comment (JohnSundell) [inline main.swift]: How about we call this `dateString` or something instead? I find it a bit more descriptive than `today` 🙂

Comment (JohnSundell) [inline main.swift]: Also why does this need to be stored here? Can't it just be inline computed in `StringReplacer`?

Comment (vknabel) [inline main.swift]: Yes, `dateString` should be better. I just wanted to treat it the same as `year`

**Comment by vknabel** [comment]
@JohnSundell Sorry for the late updates!

**Comment by gperdomor** [comment]
can we merge this?

**Comment by JohnSundell** [comment]
Sure thing, should've done this earlier, sorry about that 🙂

**Pull request merged**